libpromises/mod_common.c

```diff
@@ -492,6 +492,7 @@ const ConstraintSyntax CF_COMMON_BODIES[] =
     ConstraintSyntaxNewString("if", "", "Extended classes ANDed with context", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewString("unless", "", "Negated 'if'", SYNTAX_STATUS_NORMAL),
     ConstraintSyntaxNewStringList("meta", "", "User-data associated with policy, e.g. key=value strings", SYNTAX_STATUS_NORMAL),
+    ConstraintSyntaxNewString("promiser_attribute", "", "Specify the promiser as an attribute", SYNTAX_STATUS_NORMAL),
```

Comment:
This I don't like at all, too big of a hack. :-)

I think the pull request would work fine if that was not like that, right?